content/Unreal Tournament 2004/Mutators/R/4/b/d52473/replaces-most-of-the-weapons-and-their-ammo-with-brand-new-toys-and-ammo_[4bd52473].yml

```diff
@@ -0,0 +1,32 @@
+--- !<MUTATOR>
+contentType: "MUTATOR"
+firstIndex: "2023-03-15 23:53"
+game: "Unreal Tournament 2004"
+name: "Replaces most of the weapons and their ammo with brand new toys! (and ammo)"
+author: "Unknown"
+description: ""
```

Comment:
description: "Replaces most of the weapons and their ammo with brand new toys! (and ammo)"